src/services/ledger/index.ts

```diff
@@ -34,7 +34,6 @@ import * as adminLegacy from "./admin-legacy"
 import { MainBook, Transaction } from "./books"
 import * as caching from "./caching"
 import { TransactionsMetadataRepository } from "./services"
-import { intraledger } from "./intraledger"
```

Comment:
I can't find where the equivalent methods were added to the `LedgerFacade` shouldn't they be there somewhere?

Reply:
We were able to use the existing `recordIntraLedger` facade method for these ([here](https://github.com/GaloyMoney/galoy/pull/1408/files#diff-f0d90cf90df7bfe5ec212dae3271302b4cf75cd29fcb9e5ebd45a18c8edbbf8bR301)). The new Facade-related work was adding the metadata objects to pass into that method ([here](https://github.com/GaloyMoney/galoy/pull/1408/files#diff-6b59384ce65eb6321c50d1a3f9db34adb5b880d3b5f89e77a4de4c8838bd91c0R239))